anaconda.py

```diff
@@ -262,7 +262,7 @@ def setup_environment():
     # do this early so we can set flags before initializing logging
     from pyanaconda.flags import flags
     from pyanaconda.core.kernel import kernel_arguments
-    (opts, depr) = parse_arguments(boot_cmdline=kernel_arguments)
+    (opts, removed_no_inst_args) = parse_arguments(boot_cmdline=kernel_arguments)
```

Comment:
Is this correct? There will be no more deprecated arguments?

Reply:
It wasn't used for anything else than just `inst.` prefix so I don't see a reason to keep it universal anymore. My next step will be to remove this completely.